docs/tex/secciones/05_implementacion.tex

```diff
@@ -299,6 +304,22 @@ \subsubsection{Recomendaciones de los usuarios}
 
 Mediante algo de lógica se controla fácilmente la creación de recomendaciones, adición de comentarios a recomendaciones existentes y votaciones, los cuales se pueden ver fácilmente en \href{https://github.com/Anglepi/My-Many-Reads/blob/main/mmr-backend/user_recommendation.py}{este fichero} y en \href{https://github.com/Anglepi/My-Many-Reads/blob/main/mmr-backend/api.py}{la api}.
 
+\subsubsection{Recomendaciones automáticas}
+
+Para elaborar un algoritmo que pueda hacer sugerencias de manera automática para los usuarios, es necesario en primer lugar ser capaz de determinar los gustos de dicho usuario. Para ello, dado que un usuario puede organizar sus lecturas en varias bibliotecas, se van a generar las recomendaciones en base a los gustos definidos en una biblioteca específica, aportando mayor versatilidad a los usuarios de cara a la búsqueda de nuevas lecturas.
```

Comment:
¿Varias bibliotecas o estanterías? ¿Qué llamas exactamente una biblioteca?

Reply:
En el contexto de esta aplicación, una biblioteca es un conjunto de libros identificado por un nombre y perteneciente a un usuario. Juraría que está más que explicado en alguna de las secciones que hice muy al principio de empezar el TFM

Reply:
Pero ¿un usuario puede tener varias bibliotecas? ¿De acuerdo con qué criterio?

Reply:
Quizás quiera separar libros que lee por ocio de libros que lee por formación o interés cultural

Reply:
¿Qué historia de usuario es esa?

Reply:
No está explícito así literal, pero es parte de #30, referenciando la parte de gestionar mis lecturas. Gestionar lecturas no solo implica "Me he leído esto y le pongo tal nota", también incluye el concepto que creo que tienes tu de estanterías, que puedes crear varías en función de lo que quieras.

Reply:
Sería mucho más claro si hablaras de *several libraries* o *shelves in a library*. La palabra "biblioteca" no tiene un significado sin ambigüedad en este concepto, y conviene que lo hagas un poco más explícito en la línea que has puesto arriba en la HU.

Reply:
#74 para esto, lo meteré en #65